src/index.html

```diff
@@ -13,5 +13,158 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action=" https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="formField">
+          <label for="surname">Surname:</label>
+          <input
+           type="text"
+           name="surname"
+           id="surname"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
          >
```
Везде должно быть два отступа, у тебя не везде два отступа, писал об этом в предыдущем ревью